provider-directory/pom.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>provider-directory-parent</artifactId>
+    <groupId>gov.va.api.health</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>provider-directory</artifactId>
```

Comment:
This temporary pom doesn't really match the `provider-directory-nginx-proxy` artifact we're going to have. Let's remove this; it is valid to have a parent pom with an empty list of modules.